orgchart.md

```diff
@@ -76,7 +76,11 @@ SIP is our instant-response team whenever something goes wrong, is about to go w
 
 ### QA
```

Comment:
Since you cover accessibility more, you might want to change the section heading:

```suggestion
### QA & Accessibility
```